tests/ttls

```diff
@@ -38,11 +38,14 @@ run_test() {
     CERT="$2"
     SRV_ARGS=$3
     CLNT_ARGS=$4
-    expect -c "spawn $CHECKER openssl s_server -accept \"${PORT}\" -naccept 1 -key \"${KEY}\" -cert \"${CERT}\" $SRV_ARGS;
-        set timeout 60;
+
+    export PKCS11_PROVIDER_DEBUG="file:${TMPPDIR}/p11prov-debug-server.log"
```

Comment:
Good point. I will add it. It was mostly for debugging to be able to figure out what is going on there while working on this issue. It will fill with all the subsequent logs from all consequent servers, but at least it will not try to interleave with the client.

Reply:
Resolved. Created separate for tls-client and tls-server.